packages/openapi-generator/src/generator.ts

```diff
@@ -75,10 +74,23 @@ async function generateSDKSources(
   openApiDocument: OpenApiDocument,
   options: GeneratorOptions
 ): Promise<void> {
+  // TODO: This isn't really "request builder" anymore
   logger.info(`Generating request builder in ${serviceDir}.`);
   // TODO: what about overwrite?
+  if (openApiDocument.schemas.length) {
+    const modelDir = resolve(serviceDir, 'model');
+    await createInterfaceFiles(modelDir, openApiDocument);
+    await createFile(
+      modelDir,
+      'index.ts',
+      modelIndexFile(openApiDocument),
+      true
```

Comment:
General: Should the overwrite flag not be taken from the options?

Reply:
Yes, but it is not implemented yet.